projects/igniteui-angular/src/lib/grids/cell.component.html

```diff
@@ -32,9 +32,9 @@
             formatter
                 ? (value | columnFormatter: formatter)
                 : column.dataType === "number"
-                ? (value | igxdecimal: grid.locale)
+                ? (value | number: undefined:grid.locale)
```

Comment:
Don't pass `undefined` as format argument, just expose a new input for this in the grid column.